### PR TITLE
DB-10186 Add SYSIBM.SYSINDEXES and SYSIBM.SYSINDEXCOLUSE views

### PR DESCRIPTION
**Pull request by ascend1**

### CODE REVIEW

**Comment by ascend1** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Review by msirek — APPROVED** [review]

**Review by carolp-503 — CHANGES_REQUESTED** [review]
Zhen, changes look good, but can you add a couple of ITs that select from these so we have automatic basic function testing

**Comment by ascend1** [comment]
> Zhen, changes look good, but can you add a couple of ITs that select from these so we have automatic basic function testing

Sure, an IT is added in the latest commit.

**Comment by ascend1** [comment]
jenkins please test branch @cdh6.3.0

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

File: splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java [hunk 1]
```diff
@@ -81,6 +81,7 @@ else if(v1==v2)
         scripts.put(new Splice_DD_Version(sdd,3,2,0, 1983), new UpgradeScriptToAddBaseTableSchemaColumnsToSysTablesInSYSIBM(sdd,tc));
         scripts.put(new Splice_DD_Version(sdd,3,2,0, 1985), new UpgradeScriptToAddSysNaturalNumbersTable(sdd, tc));
         scripts.put(new Splice_DD_Version(sdd,3,2,0, 1989), new UpgradeScriptToAddIndexColUseViewInSYSCAT(sdd, tc));
+        scripts.put(new Splice_DD_Version(sdd,3,2,0, 1992), new UpgradeScriptToAddSysIndexesViewInSYSIBMAndUpdateIndexColUseViewInSYSCAT(sdd, tc));
     }
```
Comment (jyuanca) [inline splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java:85]: This PR is unlikely to merge to 1992 build. Change to 1993 for next week's build

**Review by jyuanca — CHANGES_REQUESTED** [review]

File: splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java (hunk 1 above)
Comment (ascend1) [reply splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java]: Sure, it's done together with merging master in and resolving conflicts in this file.

**Comment by ascend1** [comment]
jenkins please test branch @cdh6.3.0

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/2228/

**Review by carolp-503 — APPROVED** [review]
+1

**Comment by ascend1** [comment]
jenkins please test branch @cdh6.3.0

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Review by dgomezferro — APPROVED** [review]

**Review by jyuanca — APPROVED** [review]

**Pull request merged**